boa_engine/src/vm/code_block.rs

```diff
@@ -177,7 +177,7 @@ impl CodeBlock {
         let opcode: Opcode = self.code[*pc].try_into().expect("invalid opcode");
         *pc += size_of::<Opcode>();
         match opcode {
-            Opcode::RotateLeft | Opcode::RotateRight => {
+            Opcode::RotateLeft | Opcode::RotateRight | Opcode::SetFunctionName => {
```

Comment:
```suggestion
            Opcode::SetFunctionName => {
                  let operand = self.read::<u8>(*pc);
                  *pc += size_of::<u8>();
                  match operand {
                     0 => "prefix: none",
                     1 => "prefix: get",
                     2 => "prefix: set",
                  } 
            }
            Opcode::RotateLeft | Opcode::RotateRight  => {
```

I think it would be good to print the prefix, instead of a number